### DISCUSSION TITLE
Walls taking twice as much damage

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13967] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It seems like walls are either taking twice as much damage or their health is set to half of what its supposed to be.
Submarine walls have 200 max health, handcannon rounds deal 20 structure damage so 10 shots would be required to fully destroy a wall. A single shot however deals twice as much damage and only 5 shots are enough to fully destroy a wall.
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/4168f192-7daa-4af8-bf4f-7f202c7813ed)
Tested with other different weapons, creature attacks and explosion command on walls and the damage inflicted was always double.
Changing the max health of the wall doesn't seem to make any difference.

### Reproduction steps

1. Shoot a submarine hull/wall section with the handcannon five times

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Created a ticket: #13990.

**Comment by itchyOwl** [comment]
Fixed in FakeFishGames/Barotrauma-development@37ca0fd.

**Comment by Jasontti** [comment]
Tested the fix and no issues found.
